mimic/test/test_nova.py

```diff
@@ -71,14 +73,14 @@ def create_server(helper, name=None, imageRef=None, flavorRef=None,
             data['metadata'] = metadata
         if diskConfig is not None:
             data["OS-DCF:diskConfig"] = diskConfig
-        body = json.dumps({"server": data})
+        body = json.dumps({"server": data}).encode("utf-8")
 
     create_server = request_func(
         helper.test_case,
         helper.root,
         b"POST",
         '{0}/servers'.format(helper.get_service_endpoint(
-            "cloudServersOpenStack", region)),
+            "cloudServersOpenStack", region)).encode("ascii"),
```

Comment:
To be honest, this decision was somewhat arbitrary.  However, all these URLs are more or less guaranteed to be ASCII so it would be good to explode if something started surprising us by inserting non-ASCII code points in there.